Neos.Flow/Tests/Unit/ObjectManagement/Proxy/ProxyClassTest.php

```diff
@@ -23,16 +24,20 @@ class ProxyClassTest extends UnitTestCase
      */
     public function proxyClassesDataProvider()
     {
+        require_once(__DIR__ . '/../Fixture/ClassWithoutNamespace.php');
+
         return [
             [
-                'originalClassName' => '\Acme\Namespace\ClassName',
+                'originalClassName' => ClassImplementingInterfaceWithConstructor::class,
                 'originalClassAnnotations' => [],
                 'originalClassDocumentation' => '',
                 'originalClassConstants' => [['name' => 'TEST_CONSTANT', 'value' => '1']],
                 'expectedProxyCode' =>
```

Comment:
maybe one could use heredoc but since this is working now fine already ...

  ```
                  <<<PHP
                    class ClassImplementingInterfaceWithConstructor extends ClassImplementingInterfaceWithConstructor$compilerOriginalClassnameSuffix implements \\Neos\\Flow\\ObjectManagement\\Proxy\\ProxyInterface
                    {
                        const TEST_CONSTANT = 1;
                    }
                    PHP
```

Reply:
I don't want to touch too much code, as I think this PR is big enough already. To be honest, I'm also running out of time a bit before the feature freeze.